plugins/modules/nxos_interface_ospf.py

```diff
@@ -16,19 +16,15 @@
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-ANSIBLE_METADATA = {
-    "metadata_version": "1.1",
-    "status": ["preview"],
-    "supported_by": "network",
-}
-
 
-DOCUMENTATION = """module: nxos_interface_ospf
+DOCUMENTATION = """
+module: nxos_interface_ospf
 extends_documentation_fragment:
 - cisco.nxos.nxos
 short_description: Manages configuration of an OSPF interface instance.
```

Comment:
@NilashishC I believe for short description update u had diff PR, correct?

Reply:
Not sure which one you're taking about.

Reply:
ignore it, I checked for resource modules and it’s as expected